### ISSUE TITLE
Enter a one-line summary of the issue

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#879]
[Originally posted on GoogleCode (id 493) on 2011-05-16]

<b>(This is the template to report a data issue. If you want to report</b>
<b>an interface issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/4328

<b>What data are incorrect or missing?</b>
Minuartia rubella (Walhenberg) Hieronymus

<b>What data are you expecting instead?</b>
Minuartia rubella (Walhenberg) Hieron
Also, is Minuartia verna a definite synonym or should it be M. verna auct. non?  We have a note in our NF checklist that says: Minuartia rubella (Walhenberg) Hieronymus.

<b>If applicable, please provide an authoritative source.</b>
Tropicos, IPNI authors

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2011-05-19 14:17Z]

Sue
There is no author called Hieron. Why do you want to change the correct Hieronymus to Hieron?
With respect to M. verna as a syn: the issue is unclear. FNA lists 2 vars. of Arenaria verna as syn of M. rubella, but not A. verna itself (or the typical var.) This requires more consideration.

**Comment by sjmeades** [comment]
[Originally posted on GoogleCode on 2011-09-13 16:17Z]

Hi Luc: This was a typo on my part, I meant to write: 
it should be Minuartia rubella (Wahlenberg) Hiern [without the 'o'], rather than (Wahlenberb) Hieronymus
Tropicos, IPNI, CAA, and USDA all have Hiern, although USDA added a period after 
Hiern, making it look like an abbreviation.

See: http://www.tropicos.org/Name/6300168

IPNI authors has Hiern  - William Philip Hiern 1839-1925

Also, I just noticed that the basionym author Wahlenberg, is misspelled as 
Walhenberg in the following 3 combinations:

Accepted name: Minuartia rubella (Walhenberg) Hieronymus - should be .... (Wahlenberg) Hiern
Synonym: Alsine hirta var.rubella (Walhenberg) Hartman - should be ...(Wahlenberg) Hartman
Synonym: Alsine rubella Walhenberg - should be ... Wahlenberg

**Comment by sjmeades** [comment]
[Originally posted on GoogleCode on 2011-09-13 16:22Z]

Hi Luc: This was a typo on my part, I meant to write: 
it should be Minuartia rubella (Wahlenberg) Hiern [without the 'o'], rather than (Wahlenberb) Hieronymus
Tropicos, IPNI, CAA, and USDA all have Hiern, although USDA added a period after 
Hiern, making it look like an abbreviation.

See: http://www.tropicos.org/Name/6300168

IPNI authors has Hiern  - William Philip Hiern 1839-1925

Also, I just noticed that the basionym author Wahlenberg, is misspelled as 
Walhenberg in the following 3 combinations:

Accepted name: Minuartia rubella (Walhenberg) Hieronymus - should be .... (Wahlenberg) Hiern
Synonym: Alsine hirta var.rubella (Walhenberg) Hartman - should be ...(Wahlenberg) Hartman
Synonym: Alsine rubella Walhenberg - should be Alsine rubella Wahlenberg

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2011-09-14 13:24Z]

Luc: Regarding Minuartia rubella (issue 493).  I think it might be clearer if I described 
the problem here as well as on the form, since that issue is marked closed/won't fix.  
I wasn't sure you'd see any recent changes.  Thanks, Sue.

This was a typo on my part, I meant to write: 
it should be Minuartia rubella (Wahlenberg) Hiern [without the 'o'], 
rather than (Wahlenberb) Hieronymus
Tropicos, IPNI, CAA, and USDA all have Hiern, although USDA added a period after 
Hiern, making it look like an abbreviation.

See: http://www.tropicos.org/Name/6300168

IPNI authors has Hiern  - William Philip Hiern 1839-1925

Also, I just noticed that the basionym author Wahlenberg, is misspelled as 
Walhenberg in the following 3 combinations:

Accepted name: Minuartia rubella (Walhenberg) Hieronymus - should be .... (Wahlenberg) Hiern
Synonyms: Alsine hirta var.rubella (Walhenberg) Hartman - should be ...(Wahlenberg) Hartman
	  Alsine rubella Walhenbert - should be ... Wahlenberg
